codersdk/deployment.go

```diff
@@ -710,6 +711,15 @@ when required by your organization's security policy.`,
 			Group:       &deploymentGroupNetworkingDERP,
 			YAML:        "relayURL",
 		},
+		{
+			Name:        "Disable Direct Connections",
+			Description: "Disable peer-to-peer (aka. direct) workspace connections. All workspace connections from the CLI will be proxied through Coder (or custom configured DERP servers) and will never be peer-to-peer when enabled. Workspaces may still reach out to STUN servers to get their address until they are restarted after this change has been made, but new connections will still be proxied regardless.",
+			Flag:        "disable-direct",
```

Comment:
I think disable-direct-connections would be a bit easier to understand in isolation.

Reply:
`disable-direct-connections` is going to be used as a global flag in the CLI in another PR, so I'm changing this to `block-direct-connections`.